### PR TITLE
Support v-prediction checkpoint models when set in the model manager

### LINKED ISSUE #6268: [bug]: Extremely low image quality on some models
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Windows

### GPU vendor

Nvidia (CUDA)

### GPU model

GTX 1070

### GPU VRAM

8GB

### Version number

4.1.0

### Browser

Chrome 124.0.6367.61

### Python dependencies

{
  "accelerate": "0.29.2",
  "compel": "2.0.2",
  "cuda": "12.1",
  "diffusers": "0.27.2",
  "numpy": "1.26.4",
  "opencv": "4.9.0.80",
  "onnx": "1.15.0",
  "pillow": "10.3.0",
  "python": "3.11.7",
  "torch": "2.2.2+cu121",
  "torchvision": "0.17.2+cu121",
  "transformers": "4.39.3",
  "xformers": "0.0.25.post1"
}

### What happened

When using some models, image quality is incredibly poor. The models I have encountered this with worked on a previous version of InvokeAI fine (unsure of version, definitely 3.x). However, I reinstalled and updated to the latest version today and these models stopped functioning properly.
Example images provided both had a prompt of solely 'fox, portrait' at 512x512 using DPM++ 2M Karras at 40 steps, CFG 7.5, default VAE.
The first is using easyfluff_5000DefaultvaeFFR3CS2 (easily searched for as 'easyfluff'), the second is using yiffymix_v43. Both were pointed at their appropriate config file and checked using both epsilon and v_prediction, both were also tested as both a checkpoint and a diffusers model.
image
image

### What you expected to happen

I had expected these models to work as they previously did. For example, this is one using the yiffymix_v43 two days ago.
image

### How to reproduce the problem

Using version 4.1.0, download either yiffymix_v43 or easyfluff_5000DefaultvaeFFR3CS2 and attempt to generate any image at all.

### Additional context

I should note that using any of the starter models listed in this version of invoke, such as Deliberate_v5, works absolutely fine. I have also redownloaded all the models and associated files that are having this issue to ensure they haven't mysteriously become corrupted or somesuch at the same time as the update.

### Discord username

jantdarvus

### LINKED ISSUE #7495: [bug]: SDXL based V-pred models are treated as epsilon prediction - result noisy images
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Windows

### GPU vendor

Nvidia (CUDA)

### GPU model

RTX 3080 12GB

### GPU VRAM

12GB

### Version number

5.5

### Browser

Invoke Community Edition v5.5 launcher / MS Edge 131.0.2903.99 (Official build) (64-bit)

### Python dependencies

accelerate==1.0.1

compel==2.0.2

cuda==12.4

diffusers==0.31.0

numpy==1.26.3

opencv==4.9.0.80

onnx==1.16.1

pillow==10.2.0

python==3.11.11

torch==2.4.1+cu124

torchvision==0.19.1+cu124

transformers==4.46.3

xformers==Not Installed

### What happened

SDXL based v_prediction models are assigned epsilon prediction type automatically even when vpred and zsnr state_dict keys are present in the model, and manually changing prediction type to v_prediction is not respected. This results in unusably noisy outputs from these models.
b6c91ac0-aad5-4427-a9e6-a879df825bef

### What you expected to happen

v_prediction based models should have the correct prediction type detected based on the state_dict keys within the model metadata. In the event that this fails due to missing keys or any other reason, the user's manually selected prediction type under model settings should be respected resulting in normal quality outputs.

### How to reproduce the problem

1. Download the V-Pred-1.0-Version of this model [noobai-xl-nai-xl](https://civitai.com/models/833294/noobai-xl-nai-xl)
2. Add the model through Invoke-AI's model management ui
3. Model will be detected as an epsilon prediction model
4. Change prediction type manually to v_prediction
5. Generate image with the downloaded model
6. The output will be extremely noisy and low quality

### Additional context

I also attempted converting the model to Diffusers format both before and after manually setting the prediction type with no change in results. Additionally, the option to enable zsnr does not seem to exist in Invoke-AI though that seems to be a missing feature rather than a bug.

### Discord username

bwm_nubby

### PR DESCRIPTION
**Pull request by dunkeroni**
## Summary

Capture prediction type from model settings and pass it to the scheduler to enable v_prediction checkpoints to work again.

## Related Issues / Discussions

I'm certain there is a more correct way to handle this in the model manager that would also allow automatic detection from checkpoint files, but this at least gets vpred models working again before we go make sweeping changes to the model probe code.

Since our conversion code does not check the settings on the checkpoint, attempting to convert a v_prediction model to Diffusers will still default over to epsilon prediction and it will forever fail afterwards. It is not currently possible for users to change a diffusers model to v_prediction in Invoke without editing the local json files. User Mewt on discord has found that huggingface repos with correct scheduler_config.json are able to work without issues, but I have found others online that still specify epsilon. 

Closes #7495 
Closes #6268 (edit by @psychedelicious to get GH to pick up that this PR closes this issue)

## QA Instructions

Before:
image

After:
image

## Checklist

- [x] _The PR has a short but descriptive title, suitable for a changelog_
- [ ] _Tests added / updated (if applicable)_
- [ ] _Documentation added / updated (if applicable)_
- [ ] _Updated `What's New` copy (if doing a release after this PR)_

### CODE REVIEW

**Comment by GrettaKlu** [comment]
Not sure  what the right way of doing this is, but the changes here error when using the upscale tab. The file "tiled_multi_diffusion_denoise_latents.py" on line 216 references the get_scheduler() function and needs the unet_config argument passed through as well.

**Comment by RyanJDick** [comment]
I added a fix for the Multi-Diffusion inference issue in [160a36a](https://github.com/invoke-ai/InvokeAI/pull/7504/commits/160a36a0bfaae61092f80bcb772de62810f1dc25).

I tested the following...

Setup:
1. Install `noobaiXLNAIXL_vPred10Version` model
2. The prediction_type gets probed incorrectly as epsilon (this will have to be addressed in a separate PR)
3. Change the model's prediction_type in the models tab.

Tests:
- [x] Generate with the noobai model (v-pred), and verify that it produces coherent results
- [x] Generate with an epsilon-pred model and verify that the output is the same as before this change.
- [x] Upscale with the noobai model and verify that it works in a Multi-Diffusion pipeline
- [x] Upscale with an epsilon-pred model and verify that the output is the same as before the change.

**Review by RyanJDick — APPROVED** [review]

**Comment by psychedelicious** [comment]
I believe this also closes #6268.

**Review by hipsterusername — APPROVED** [review]
